### ISSUE TITLE
Double quotes are escaped and included in string

### ISSUE THREAD

**Issue by GameFuzzy** [Jelmerro/Vieb#556]
**Checklist**
- [x] I searched the `:help` documentation using `/` for helpful information
- [x] I have read the [frequently asked questions](https://github.com/Jelmerro/Vieb/blob/master/FAQ.md)
- [x] I did not find any similar issues in the [issue tracker](https://github.com/Jelmerro/Vieb/issues)
- [x] I prefer making an issue instead of using [discussions](https://github.com/Jelmerro/Vieb/discussions), [Matrix](https://matrix.to/#/#vieb:matrix.org) or [Telegram](https://t.me/vieb_general)

**Describe the bug**
When trying to set a multiword string as described in the docs below the double quotes are instead escaped and included in said string.
![image](https://github.com/Jelmerro/Vieb/assets/30367459/0a4197ed-c163-450e-8f78-be4170835cd1)

**To Reproduce**
`:set vimcommand="gnome-terminal -- vim"`

**Expected behavior**
The string should look like this `gnome-terminal -- vim`

**Extra information**
Should probably also add that I get this error whenever i run `vieb` in case it's somehow related:
```
2024-04-12 23:42:54.598 Vieb[88298:139493430] WARNING: Secure coding is automatically enabled for restorable state! However, not on all supported macOS versions of this application. Opt-in to secure coding explicitly by implementing NSApplicationDelegate.applicationSupportsSecureRestorableState:.
Sending urls to existing instance /Users/fuzzy/Library/Application Support/Vieb/
```
Assuming this bug is exclusive to MacOS since it probably would've been reported already otherwise.

**Specs**
 - OS: MacOS Sonoma 14.4.1
 - Version: Vieb 11.0.0

**Comment by Jelmerro** [comment]
Thank you for reporting this issue. This is a big problem, and one that needs fixing, but it's once again a breaking change, so I'm taking this opportunity to change some other things for the big 12 release now. It will probably mean that only double quotes will remain working, as this is in line with JSON that's also used for object and array parsing, which caused string escaping to not work exactly the same as before.

As for the startup error, these are Electron/Chromium related, and not in any way related to Vieb's code, otherwise they would be in the `:internaldevtools`, also see `--devtools`. Those logs are from the main process, which basically just handles setting up the (Electron) window and responding to internal messages.

**Comment by Jelmerro** [comment]
Implemented in 12.0.0 :tada:
